content/topics/solo-learn/introduction-to-sql/1-getting-started-with-sql/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+content_type: topic
+ready: true
+tags:
+- sql
+- solo-learn
+title: Solo Learn - Introduction to SQL - 1 Getting Started with SQL
+---
+
+Complete the above-mentioned module on Solo Learn.
+
+N.B. When signing up for the course ensure that you sign up for [Introduction to SQL](https://www.sololearn.com/en/learn/courses/sql-introduction)
+
+Ensure you check your own profile to ensure that it shows that you have started the [Introduction to SQL](https://www.sololearn.com/en/learn/courses/sql-introduction) course.
+
+Please submit the link to your SoloLearn profile, for example, https://www.sololearn.com/profile/21320495
+
+We just need to make sure everyone is keeping up 💚
+
+## Instructions for the reviewer
+
+- Please ensure that the learner submitted a link to their Solo Learn Profile.
+- Please ensure that the Introduction to SQL course is "In Progress".
```

Comment:
The content for this should not have anything to do with **submitting anything** or **reviewing instructions** since it's a topic and we're not checking anything.